### PR TITLE
Sort out Python imports and docker files

### PR DESCRIPTION
**Pull request by mabe13**
This change sorts out the python imports and docker compose files for the two python applications `data to file conversion` and `file to image conversion`.

**Note** however the both application docker image don't run without error as both need to be pointed to Localstack or AWS to pass thwie infrastructure dependencies. The `file to image conversion` job also needs to be refacted so it doesn't push to Azure and also create the dataload image.

### CODE REVIEW

File: application/data_to_file_conversion/tests/__init__.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import sys, os
```
Comment (JackPlowman) [inline application/data_to_file_conversion/tests/__init__.py:1]: sys and os should be removed if not used

File: build/docker/file-to-image-conversion/Dockerfile.effective [hunk 2]
```diff
@@ -0,0 +1,34 @@
+FROM nhsd/python:20200614
```
Comment (JackPlowman) [inline build/docker/file-to-image-conversion/Dockerfile.effective:1]: This file should be a Dockerfile and the Dockerfile.effective should be ignored

File: build/docker/file-to-image-conversion/Dockerfile.effective [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ADD assets/app.tar.gz /application/file_to_image_conversion
+
+WORKDIR /
+RUN python -m pip install --upgrade pip && python -m pip install -r /application/file_to_image_conversion/requirements.txt
+
+RUN apk add postgresql-client
+
+ENTRYPOINT [ "/sbin/entrypoint.sh" ]
+
+### METADATA ###################################################################
```
Comment (JackPlowman) [inline build/docker/file-to-image-conversion/Dockerfile.effective:13]: If this file should be a Dockerfile then the Metadata isn't required and should be removed

File: application/file_to_image_conversion/tests/__init__.py [hunk 4]
```diff
@@ -1,4 +1,3 @@
 import sys, os
```
Comment (JackPlowman) [inline application/file_to_image_conversion/tests/__init__.py:1]: sys and os aren't required so can be removed

File: application/data_to_file_conversion/tests/__init__.py (hunk 1 above)
Comment (mabe13) [reply application/data_to_file_conversion/tests/__init__.py:1]: Will remove

File: application/file_to_image_conversion/tests/__init__.py (hunk 4 above)
Comment (mabe13) [reply application/file_to_image_conversion/tests/__init__.py:1]: Will remove

File: build/docker/file-to-image-conversion/Dockerfile.effective (hunk 2 above)
Comment (mabe13) [reply build/docker/file-to-image-conversion/Dockerfile.effective:1]: This effective docker file needs to be delete, a new docker file was added see here  https://github.com/nhsd-ddce/pu-data-load-process/pull/16/files#diff-fd5bed7ea0c76b8a2fed3a630833f413

File: build/docker/file-to-image-conversion/Dockerfile.effective (hunk 3 above)
Comment (mabe13) [reply build/docker/file-to-image-conversion/Dockerfile.effective:13]: This effective docker file needs to be delete, a new docker file was added see here  https://github.com/nhsd-ddce/pu-data-load-process/pull/16/files#diff-fd5bed7ea0c76b8a2fed3a630833f413

Comment (mabe13) [reply build/docker/file-to-image-conversion/Dockerfile.effective:13]: Accidently delete @JackPlowman comment on this one but basically the effective dockerfile needs to be added to ignore and the actual dockerfile added

**Pull request merged**